SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -761,8 +760,6 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 
 			if (bIsRecentlyRelevant)
 			{
-				FinalRelevantCount++;
```

Comment:
I think we want to keep this in?

Reply:
Oh yeah, think I got myself confused there too. Does look like FinalRelevantCount is not actually used. Will remove it 👍